src/PaletteProvider.js

```diff
@@ -93,11 +100,48 @@ export default class MaterialPaletteProvider
     };
   }
 
+  _mergeWithDefaults(palette: PaletteInstance) {
+    const defaults = {
+      ...defaultSwatches,
+      ...Object.keys(this.props.defaults || {}).reduce(
+        (acc: *, profile: string) => ({
+          ...acc,
+          [profile]: { ...(this.props.defaults[profile] || {}), population: 0 },
+        }),
+        {},
+      ),
+    };
+    return {
+      ...Object.keys(palette)
```

Comment:
You don't need to spread the object inside object -> you're reducing it to object already.